.readthedocs.yaml

```diff
@@ -0,0 +1,20 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+   configuration: docs/conf.py
+
+# Optionally build your docs in additional formats such as PDF
+formats:
+   - pdf
+
+# Optionally set the version of Python and requirements required to build your docs
+python:
+   version: 3.8
+   install:
+   - requirements: docs/requirements.txt
```

Comment:
Um, this PR deletes that file. Is it needed?

Reply:
nope

Reply:
`requirements.txt` is not deleted?

Reply:
Okay, true. But it's empty. Is that intentional?

Reply:
You're right that it's not strictly necessary but if we want to add dependencies in the future i dont think there's any harm in leaving it?